docs/requirements.txt

```diff
@@ -1,5 +1,4 @@
 # This file is intended for use on readthedocs
-sphinx >= 3.0
+sphinx
 sphinx-rtd-theme
-robotpy-sphinx-plugin
--e docs
\ No newline at end of file
+robotpy-sphinx-plugin~=0.1,>=0.1.6
```

Comment:
Isn't this equivalent to `robotpy-sphinx-plugin~=0.1.6`?

Reply:
That appears to be the case. I haven't used them much.